RELEASING.md

```diff
@@ -1,16 +1,15 @@
 #### (Somewhat) automated process: 
 1. Create a branch bump/x.y.z
+1. Create a CHANGELOG.latest.md with the changes for the current 
+version (to be used by Fastlane for the github release notes)
 1. Update the version number in `RCPurchases.m`, `Purchases.podspec` and in 
-`Purchases/Info.plist` by running `fastlane bump version:x.y.z`
-1. Update CHANGELOG.md for the new release
+`Purchases/Info.plist` by running `fastlane bump_and_update_changelog version:x.y.z`
```

Comment:
Just realized. Should this be the same in the Manual process??

Reply:
good question... I wanted the manual process to be sort of a backup in case automations fail. like, maybe it shouldn't even reference the `fastlane bump` command. I'm merging this version for now, but let's keep the conversation open.